RELEASE_NOTES

```diff
@@ -1,3 +1,8 @@
+# Version 1.0.16 (2017-12-20)
+
+- Issue #9950 - VADCP  missing expected parameters and failed creation of true upward velocity
+    * Added parameter PD3822 (error_seawater_velocity) in stream "vadcp_velocity_beam
+
```

Comment:
Add a comment explaining that previous changes to ParameterDefs.csv, adding "Science Data" to the temperature parameters' dataproducttype column, were not updated in preload_database.sql.  Those changes are being committed in this issue.